### PR TITLE
Add ActorMethodInfoMap

### PR DESCRIPTION
**Pull request by LMWF**
# Description

Adds ActorMethodInfoMap + unit test.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_29_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: sdk/src/main/java/io/dapr/actors/runtime/ActorMethodInfoMap.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
+package io.dapr.actors.runtime;
+
+import java.lang.reflect.Method;
+import java.util.HashMap;
+
+class ActorMethodInfoMap {
```
Comment (artursouza) [inline sdk/src/main/java/io/dapr/actors/runtime/ActorMethodInfoMap.java]: comments, please.

File: sdk/src/test/java/io/dapr/actors/runtime/ActorMethodInfoMapTest.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        try {
+            Method m1 = m.LookupActorMethodInfo("getData");
+            Assert.assertEquals("getData", m1.getName());
+            Class c = m1.getReturnType();
+            Assert.assertEquals(c.getClass(), String.class.getClass());
+            Parameter[] p = m1.getParameters();
+            Assert.assertEquals(p[0].getType().getClass(), String.class.getClass());
+        }
+        catch(Exception e) {
+            Assert.fail("Exception not expected.");
```
Comment (artursouza) [inline sdk/src/test/java/io/dapr/actors/runtime/ActorMethodInfoMapTest.java]: No need to try catch it. Let the exception propagate, the test will fail as well with less code.

File: sdk/src/main/java/io/dapr/actors/runtime/ActorMethodInfoMap.java (hunk 1 above)
Comment (LMWF) [reply sdk/src/main/java/io/dapr/actors/runtime/ActorMethodInfoMap.java]: Done

File: sdk/src/test/java/io/dapr/actors/runtime/ActorMethodInfoMapTest.java (hunk 2 above)
Comment (LMWF) [reply sdk/src/test/java/io/dapr/actors/runtime/ActorMethodInfoMapTest.java]: Unfortunately that doesn't compile.  The only alternative I can think of is to mark `normalUsage()` itself with a `throws`, but that looks weird too.  I'm going to leave it, but if you know of a better way we can change it.

**Pull request merged**